libgdf/src/gdf_table.cuh

```diff
@@ -45,6 +45,30 @@ struct ValidRange {
     }
 };
 
+template<typename size_type, typename byte_type>
+__device__
+byte_type get_bit(size_type index, byte_type const * data)
```

Comment:
This functionality is already implemented here: https://github.com/rapidsai/cudf/blob/master/libgdf/include/gdf/utils.h#L10